recipes/nikola/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "nikola" %}
+{% set version = "8.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/getnikola/nikola/archive/v{{ version }}.tar.gz
+  sha256: 558c715b5df4ec065e410a084dafb96e50291f4972e378770532e9c930768610
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
You need to add all the requirements here,
https://github.com/getnikola/nikola/blob/master/requirements.txt
Conda names of these packages are usually lowercase names.

Reply:
Ugh. Ok.

Reply:
Thanks for the hint.

How can one test this, other than changing it blindly and waiting for CI? Any links? I see nothing here: https://conda-forge.org/#add_recipe

Reply:
If you like docker, you can run `.circleci/run_docker_build.sh` script.

Reply:
Nevermind, there are too many dependencies that are not packaged for conda, not going to do this.